packages/access/src/stores/store-indexeddb.js

```diff
@@ -0,0 +1,227 @@
+import { importDAG } from '@ucanto/core/delegation'
+import * as Signer from '@ucanto/principal/rsa'
+import defer from 'p-defer'
+import { Delegations } from '../delegations.js'
+
+/**
+ * @typedef {import('./types').StoreDataKeyRsa} StoreData
+ * @typedef {import('./types').StoreKeyRsa} Store
+ */
+
+const STORE_NAME = 'AccessStore'
+const DATA_ID = 1
+
+/**
+ * Store implementation for the browser.
+ *
+ * @implements {Store}
+ */
+export class StoreIndexedDB {
+  /** @type {string} */
+  #dbName
+
+  /** @type {number|undefined} */
+  #dbVersion
+
+  /** @type {IDBDatabase|undefined} */
+  #db
+
+  /**
+   * @param {string} dbName
+   * @param {number} [dbVersion]
+   */
+  constructor(dbName, dbVersion) {
+    this.#dbName = dbName
```

Comment:
Are these widely enough available for us to use ? If I recall correctly they were considered in js-multiformats but then dropped because adoption was not there yet.

Reply:
I can put underscores...

Reply:
I think it's fine!

<img width="781" alt="Screenshot 2022-10-26 at 18 19 14" src="https://user-images.githubusercontent.com/152863/198093542-2cdeb293-e417-4eed-afb4-5bfa387ce048.png">